pkg/client/client_test.go

```diff
@@ -63,6 +75,20 @@ func TestCreateKafkaMultipleTopicsServers(t *testing.T) {
 	assert.NilError(t, err)
 }
 
+func TestGetKafkaSources(t *testing.T) {
+	fakeClient := fake.FakeSourcesV1alpha1{Fake: &client_testing.Fake{}}
```

Comment:
WIth updated `setup` function you'd get both vars `fakeClient, cli` initialized.